sys-kernel/bootengine/bootengine-9999.ebuild

```diff
@@ -34,6 +34,7 @@ src_install() {
 		"${D}"/usr/lib/dracut/modules.d/30disk-uuid/disk-uuid.sh \
 		"${D}"/usr/lib/dracut/modules.d/30ignition/ignition-generator \
 		"${D}"/usr/lib/dracut/modules.d/30ignition/ignition-setup.sh \
+		"${D}"/usr/lib/dracut/modules.d/30ignition/ignition-kargs-helper \
```

Comment:
```suggestion
```
I moved this  to the latest commit and if the above suggestion is applied it will be missing